src/components/Projects/PaginationComponent.tsx

```diff
@@ -0,0 +1,80 @@
+import { useState } from 'react';
+import Project from './Project';
+import { ProjectList } from './ProjectList';
+
+const PaginationComponent = ({ items, itemsPerPage = 6 }: any) => {
+  const [currentPage, setCurrentPage] = useState(1);
+
+  const totalPages = Math.ceil(items.length / itemsPerPage);
+
+  const indexOfLastItem = currentPage * itemsPerPage;
+  const indexOfFirstItem = indexOfLastItem - itemsPerPage;
+  const currentItems = ProjectList.slice(indexOfFirstItem, indexOfLastItem);
```

Comment:
Since you are implementing a generic pagination component you should only rely on the "items" passed as props

Reply:
nice catch

Reply:
Based on this, I could give as input the <Project> component, but for now I will not
I could do it if I will make more generic the PaginationComponent and use it for the Events section too